index.bs

```diff
@@ -1563,13 +1563,14 @@ dictionary MLPadOptions {
 };
 
 partial interface MLGraphBuilder {
-  MLOperand pad(MLOperand input, MLOperand padding, optional MLPadOptions options = {});
+  MLOperand pad(MLOperand input, sequence<sequence<long>> padding,
+                optional MLPadOptions options = {});
```

Comment:
My only holdback on this is that the `padding` data itself may be unknown during graph construction time, and that having a placeholder operand will allow for the dynamic padding at an execution time. ONNX for example has this provision for the pad op.

Reply:
According to the implementation feedback (#224), the major issue is that native ML APIs, like DirectML ([`DML_PADDING_OPERATOR_DESC`](https://docs.microsoft.com/en-us/windows/win32/api/directml/ns-directml-dml_padding_operator_desc)), only support static padding data.

Reply:
This is actually a shortcoming in DirectML, which is being fixed.

Reply:
Thanks @wchao1115 for the inputs. I have passed your feedback to the original issue and proposed to keep the current version. Once we make consensus there, I'll close this PR.